scripting/cwx/item_config.sp

```diff
@@ -305,10 +305,53 @@ int EquipCustomItem(int client, const CustomItemDefinition item) {
 			SetEntProp(itemEntity, Prop_Send, "m_iAccountID", accountid);
 		}
 	}
-	
-	// TODO: implement a version that nullifies runtime attributes to their defaults
-	SetEntProp(itemEntity, Prop_Send, "m_bOnlyIterateItemViewAttributes",
-			!item.bKeepStaticAttributes);
+
+	if(!item.bKeepStaticAttributes) {
```

Comment:
Might be time to refactor the config definition to support string values for `keep_static_attrs`?  I think it's worth leaving the option

Something like:
- Numeric values = `KeepStatic_False` or `KeepStatic_True`; only toggle `m_bOnlyIterateItemViewAttributes`
- "nullify" = `KeepStatic_Nullify`; run this code

Reply:
(Previous iterations claimed "2" was for nullifying; I don't have any issues with that approach either.)

Reply:
I haven't experienced any issues by using this method to strip static attributes but sure. I'd go with `0 / 1 / nullify`.

Reply:
Addressed this in the newest commit.